mobile/function/route_list_functions.js

```diff
@@ -22,13 +23,14 @@ export const GetRouteInformation = (apiCallParameter) => {
     const [isLoading, setLoading] = useState(true);
     //make useState of data, setData to empty array
     const [data, setData] = useState([]);
+    const nav = useNavigation();
 
     //make a component called fetchData with a useCallback function of react and set it async
     //useCallback is used once an action needs to be called several times before sending it of for use
     const fetchData = useCallback(async () => {
         //api call
         const response = await fetch(
-            `http://10.232.13.234:3000/api/routes/${apiCallParameter}`
+            `http://10.232.15.11:3000/api/routes/${apiCallParameter}`
```

Comment:
Ik zou een globale variabel maken like in een .env bestand waar het ip staat zodat je t niet overal steeds hoeft aan te passen

Reply:
even overal toevoegen dus hier ook